app/src/main/java/com/prangesoftwaresolutions/audioanchor/PlayActivity.java

```diff
@@ -385,7 +388,25 @@ void setAlbumCover() {
                 display.getSize(size);
                 reqSize = java.lang.Math.max(size.x, size.y);
             }
-            BitmapUtils.setImage(mCoverIV, mAudioFile.getCoverPath(), reqSize);
+
+            // Taken from https://stackoverflow.com/a/21549403
+            android.media.MediaMetadataRetriever mmr = new MediaMetadataRetriever();
```

Comment:
just use "MediaMetadataRetriever mmr = new MediaMetadataRetriever();"

Reply:
Obviously. Sorry! I think the latest commit of the branch should have taken care of that.